config/application.rb

```diff
@@ -37,5 +37,12 @@ class Application < Rails::Application
       g.jbuilder          false
       g.scaffold_stylesheet false
     end
+
+    # Permitted locales available for the application
+    I18n.available_locales = [:ar, :cs, :en, :es, :de, :fr, :pt, :th]
+    config.i18n.enforce_available_locales
```

Comment:
If we _do_ go w available_locales, we should enforce them. Otherwise prob don't need this?